core/src/data/value/selector.rs

```diff
@@ -53,7 +53,7 @@ impl Value {
             .try_fold(self, |selectable, key| {
                 selectable.get_value_from_compound_type(&key)
             })
-            .map(Clone::clone)
+            .cloned()
```

Comment:
nit. this also looks that is caused by clippy 1.79.

Reply:
Reverted